wordpress-mobile/WordPress-Android
# Add click tracking to unified login and signup flow

Fixes #11797

This PR adds tracking to all the clicks in the login flow. The events tracked are `unified_login_interaction` (do you think it would be better to use "click" instead?). The event contains the previous source/flow/step to know for sure what happened before in the flow. I tried to cover all the clicks by looking for all the onClickListeners but it's possible I missed some of the cases.

To test:
- Filter `unified_login_interaction` in Logcat
- Try various login flows
- Check that all the clicks are tracked and that the value tracked makes sense

PR submission checklist:
